### PR TITLE
Adding test cases for add/remove/update pool and fixing its msg_server

### PR DESCRIPTION
**Pull request by avkr003**

### CODE REVIEW

File: x/leveragelp/keeper/msg_server_add_pool_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	_, err := suite.app.AmmKeeper.CreatePool(suite.ctx, &msgCreatePool)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func (suite *KeeperTestSuite) TestAdd_Pool() {
+	suite.ResetSuite()
+	SetupCoinPrices(suite.ctx, suite.app.OracleKeeper)
+	//SetupCoinPrices(suite.ctx, suite.app.OracleKeeper, []string{ptypes.Elys, ptypes.ATOM, "uusdt"})
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_add_pool_test.go:64]:
```suggestion
```

File: x/leveragelp/keeper/msg_server_add_pool_test.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+				Pool: types.AddPool{
+					AmmPoolId:   1,
+					Enabled:     false,
+					Closed:      false,
+					LeverageMax: sdk.MustNewDecFromStr("10"),
+				},
+			},
+			expectErr:            false,
+			expectErrMsg:         "",
+			prerequisiteFunction: func() {},
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_add_pool_test.go:103]: Is it possible to add validation for new pool ?

File: x/leveragelp/keeper/msg_server_remove_pools_test.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+			panic(err)
+		}
+	}
+	suite.app.LeveragelpKeeper.SetPool(suite.ctx, types.NewPool(1))
+}
+
+func (suite *KeeperTestSuite) TestRemove_Pool() {
+	suite.ResetSuite()
+	SetupCoinPrices(suite.ctx, suite.app.OracleKeeper)
+	//SetupCoinPrices(suite.ctx, suite.app.OracleKeeper, []string{ptypes.Elys, ptypes.ATOM, "uusdt"})
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_remove_pools_test.go:35]:
```suggestion
```

File: x/leveragelp/keeper/msg_server_remove_pools_test.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+			},
+		},
+		{name: "success",
+			input: &types.MsgRemovePool{
+				Authority: "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn",
+				Id:        1,
+			},
+			expectErr:    false,
+			expectErrMsg: "",
+			prerequisiteFunction: func() {
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_remove_pools_test.go:64]: can we verify if pool was deleted ?

File: x/leveragelp/keeper/msg_server_update_pools_test.go [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+			panic(err)
+		}
+	}
+	suite.app.LeveragelpKeeper.SetPool(suite.ctx, types.NewPool(1))
+}
+
+func (suite *KeeperTestSuite) TestUpdate_Pool() {
+	suite.ResetSuite()
+	SetupCoinPrices(suite.ctx, suite.app.OracleKeeper)
+	//SetupCoinPrices(suite.ctx, suite.app.OracleKeeper, []string{ptypes.Elys, ptypes.ATOM, "uusdt"})
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_update_pools_test.go:35]:
```suggestion
```

File: x/leveragelp/keeper/msg_server_update_pools_test.go [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+				UpdatePool: &types.UpdatePool{
+					PoolId:  1,
+					Enabled: false,
+					Closed:  false,
+				},
+			},
+			expectErr:    false,
+			expectErrMsg: "",
+			prerequisiteFunction: func() {
+			},
```
Comment (amityadav0) [inline x/leveragelp/keeper/msg_server_update_pools_test.go:73]: same here

**Review by cosmic-vagabond — APPROVED** [review]

**Comment by cosmic-vagabond** [comment]
@avkr003 merging this PR now, please fix the remaining comments in a subsequent PR

**Pull request merged**